2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Configure stats pid to 128455
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Loading settings from /home/s2113174/.config/wandb/settings
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Loading settings from /home/s2113174/Projects-1/Laplace_Exp/wandb/settings
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_init.py:_log_setup():533] Logging user logs to /home/s2113174/Projects-1/Laplace_Exp/wandb/run-20250320_181804-srwsdudp/logs/debug.log
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_init.py:_log_setup():534] Logging internal logs to /home/s2113174/Projects-1/Laplace_Exp/wandb/run-20250320_181804-srwsdudp/logs/debug-internal.log
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_init.py:_jupyter_setup():479] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x785970eadca0>
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_init.py:init():619] calling init triggers
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2025-03-20 18:18:04,578 INFO    MainThread:128455 [wandb_init.py:init():644] re-initializing run, found existing run on stack: qa6s0b43
2025-03-20 18:18:04,580 INFO    MainThread:128455 [wandb_run.py:_finish():2155] finishing run christian-jb07-univeristy-of-edinburgh/PDEs-training/qa6s0b43
2025-03-20 18:18:04,580 INFO    MainThread:128455 [wandb_run.py:_atexit_cleanup():2420] got exitcode: 0
2025-03-20 18:18:04,580 INFO    MainThread:128455 [wandb_run.py:_restore():2402] restore
2025-03-20 18:18:04,580 INFO    MainThread:128455 [wandb_run.py:_restore():2408] restore done
2025-03-20 18:30:10,669 WARNING MainThread:128455 [wandb_init.py:init():1260] interrupted
Traceback (most recent call last):
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1256, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 654, in init
    latest_run.finish()
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 451, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 393, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2149, in finish
    return self._finish(exit_code)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2179, in _finish
    self._atexit_cleanup(exit_code=exit_code)
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2432, in _atexit_cleanup
    self._on_finish()
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/wandb_run.py", line 2681, in _on_finish
    _ = exit_handle.wait(
        ^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 279, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 122, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/s2113174/anaconda3/envs/fenicsx-env/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2025-03-20 18:30:10,671 WARNING MsgRouterThr:128455 [router.py:message_loop():75] message_loop has been closed
2025-03-20 18:30:12,678 INFO    MainThread:128455 [wandb_settings.py:_apply_login():1989] Applying login settings: {}
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Current SDK version is 0.18.7
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Configure stats pid to 128455
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Loading settings from /home/s2113174/.config/wandb/settings
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Loading settings from /home/s2113174/Projects-1/Laplace_Exp/wandb/settings
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_setup.py:_flush():79] Applying login settings: {}
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_init.py:_log_setup():533] Logging user logs to /home/s2113174/Projects-1/Laplace_Exp/wandb/run-20250320_183012-klos9z57/logs/debug.log
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_init.py:_log_setup():534] Logging internal logs to /home/s2113174/Projects-1/Laplace_Exp/wandb/run-20250320_183012-klos9z57/logs/debug-internal.log
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_init.py:init():619] calling init triggers
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_init.py:init():626] wandb.init called with sweep_config: {}
config: {}
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_init.py:init():669] starting backend
2025-03-20 18:30:12,683 INFO    MainThread:128455 [wandb_init.py:init():673] sending inform_init request
2025-03-20 18:30:12,685 INFO    MainThread:128455 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2025-03-20 18:30:12,685 INFO    MainThread:128455 [wandb_init.py:init():686] backend started and connected
2025-03-20 18:30:12,685 WARNING MsgRouterThr:128455 [router.py:message_loop():75] message_loop has been closed
2025-03-20 18:30:12,687 INFO    MainThread:128455 [wandb_run.py:_label_probe_notebook():1339] probe notebook
2025-03-20 18:30:12,688 INFO    MainThread:128455 [wandb_init.py:init():781] updated telemetry
